src/gwt/src/org/rstudio/studio/client/workbench/prefs/views/PackagesPreferencesPane.java

```diff
@@ -342,6 +345,25 @@ public void onError(ServerError error)
             }
          }
       );
+   
+      // If renv is managing our repos, make the info bar more specific
+      server_.readProjectOptions(
```

Comment:
Should we check first whether a project is actually open? (Or does this still do the right thing in that case as well?)

Reply:
It actually already does the right thing both 

1. in a project (with or without `renv` active), and 
2. in a non-R project (with or without `renv` active)